extensionsGallery.json

```diff
@@ -153,7 +153,160 @@
 				],
 				"statistics": [],
 				"flags": "preview"
-			}
+            },
+            {
+                "extensionId": "14",
+                "extensionName": "alwayson-insights",
+                "displayName": "AlwaysOn Insights",
+                "shortDescription": "Sql Server AlwaysOn insights",
+                "publisher": {
+                  "displayName":"matticusau",
+                  "publisherId": "matticusau",
+                  "publisherName":"matticusau"
+                },
+                "versions": [
+                  {
+                    "version": "0.2.1",
+                    "lastUpdated": "04/21/2018",
+                    "assetUri": "",
+                    "fallbackAssetUri": "fallbackAssetUri",
+                    "files": [
+                    {
+                      "assetType": "Microsoft.SQLOps.DownloadPage",
+                      "source": "https://github.com/Matticusau/sqlops-alwayson-insights/releases/tag/0.2.1"
+                    },
+                    {
+                      "assetType": "Microsoft.VisualStudio.Services.Links.Source",
+                      "source": "https://github.com/Matticusau/sqlops-alwayson-insights"
+                   },
+                   {
+                      "assetType": "Microsoft.VisualStudio.Services.Icons.Default",
+                      "source": "https://raw.githubusercontent.com/Matticusau/sqlops-alwayson-insights/master/src/images/icon.png"
+                    },
+                    {
+                      "assetType": "Microsoft.VisualStudio.Services.Content.Details",
+                     "source": "https://raw.githubusercontent.com/Matticusau/sqlops-alwayson-insights/master/src/README.md"

```

Comment:
All of the images in your README files need to be updated with absolute links or they fail to render in the Ops Studio extension gallery UI. See how it renders in my testing:

![image](https://user-images.githubusercontent.com/10819925/39088276-bd2c0e48-4563-11e8-97a8-a6ad8bf981c7.png)

It's easy enough to fix, for example your Show Changes image becomes `![Show details](https://github.com/Matticusau/sqlops-alwayson-insights/blob/master/src/images/show-detail.png?raw=true)`

Reply:
Thanks for the reminder on the image links. I remember that from my early work on a VSIX extension.
I have fixed the links in the README in an updated VSIX for each release. I kept the version number the same but if you would prefer me to officially release a patch for it I will. I have a few other things planned to so will have new versions of these in the next few weeks ;)